src/main/java/io/github/noeppi_noeppi/mods/bongo/data/GameSettings.java

```diff
@@ -133,6 +135,9 @@ public CompoundNBT getTag() {
     public void fillStartingInventory(PlayerEntity player) {
         PlayerInventory inventory = player.inventory;
         inventory.clear();
+        if (ModList.get().isLoaded("curios")) {
+            CuriosIntegration.clearTrinkets(player);
+        }
```

Comment:
If you can not get it working without curios being a compile time dependency, this should be in `DefaultEffects#playerInit` and not here.